client/dashboard/app/router/index.tsx

```diff
@@ -34,7 +35,11 @@ const createRouteTree = ( config: AppConfig ) => {
 	}
 
 	if ( config.supports.sites ) {
-		children.push( ...createSitesRoutes( config ) );
+		if ( config.context === 'ciab' ) {
```

Comment:
To me, it seems weird that we have the concept of "context" here. It then means suddenly we don't need all the config since we can just check for `context` 🥲 

Can we just add another config, e.g. `supports.stores`?

Also are we sure that the path is `/sites`? The page title is called `Stores`, and CIAB as product is about building stores, so I'm not really sure we should use `/sites` 🤔 Maybe `/stores`? Not sure. cc @taipeicoder :disappear:

Reply:
Should this be Sites or Stores

<img width="167" height="113" alt="image" src="https://github.com/user-attachments/assets/3886e5dc-e9cd-4bea-a487-90d0002d75ad" />

Reply:
It should be `stores`. Looks like it would be better to separate them into different routes instead of using the context as you suggested 😄

Reply:
I meant the path of each route. For example, if we want to change the settings URL from `/sites/:site/settings` to `/stores/:site/settings`, how would we handle that in this approach?

Reply:
Oh I misunderstood. I was referring when the routes need to be shared but that doesn't apply for the sites/stores list. We can discuss more if or when we come across that scenario.

Reply:
Also, I guess we need to update all occurrences of the term `site` in the copy to `store` in the future 🫠

Reply:
Yes, I think @p-jackson mentioned that the entire text needs to be translated due to noun gender.

Reply:
Introduced `supports.stores` config by https://github.com/Automattic/wp-calypso/pull/106182/commits/85b4d37a5f95c33c82a86c02b9c217a726c2552d. Now I keep using `/sites` instead of `/stores` as it's more complex...